kf_lib_data_ingest/etl/extract/extract.py

```diff
@@ -418,10 +442,12 @@ def _postrun_concept_discovery(self, run_output):
 
             self.logger.debug(f"Recording {config_path} links")
```

Comment:
Recording links is where the majority of wasted effort occurs, and they're not used right now, and they may never be used. Why not just trash these inner loops?

Reply:
They're not used by the accounting checks in the library but they could be used in user defined tests. I think Nick may be using them in his tests for one of his packages but I'm not sure. Should we still get rid of the loops?

Reply:
I'll ask nick, because there's a _ton_ of extra work being done in there :(

Reply:
This PR as is will also break what Nick's doing because he's not only looking at the preserved values.